testing/ncids-css-testing/stories/uswds-native/usa-table/src/usa-table.scenarios.js

```diff
@@ -0,0 +1,134 @@
+module.exports = [
+	{
+		label: 'usa-table default',
+		storyId: 'uswds-components-table--default',
+		viewports: [
+			{
+				label: 'mobile',
+				width: 479,
+				height: 360,
+			},
+		],
+	},
+	{
+		label: 'usa-table borderless',
+		storyId: 'uswds-components-table--borderless',
+		viewports: [
+			{
+				label: 'mobile',
+				width: 479,
+				height: 360,
+			},
+		],
+	},
+	{
+		label: 'usa-table segmented',
+		storyId: 'uswds-components-table--segmented',
+		viewports: [
+			{
+				label: 'mobile',
+				width: 479,
+				height: 360,
+			},
+		],
+	},
+	{
+		label: 'usa-table sortable',
```

Comment:
need focus/hover/active checks for `.usa-table__header__button` for all three variants